services/frontend-service/src/ui/Pages/CommitInfo/CommitInfoPage.test.tsx

```diff
@@ -0,0 +1,140 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+import { render } from '@testing-library/react';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { CommitInfoPage } from './CommitInfoPage';
+import { fakeLoadEverything } from '../../../setupTests';
+import { updateCommitInfo, CommitInfoState } from '../../utils/store';
+import { GetCommitInfoResponse } from '../../../api/api';
+
+/*
+    using this would be the ideal scenario for testing the CommitInfoPage component
+    but it doesn't work, any help is most appreciated.
+*/
+// const mockCommitInfo = jest.fn();
+// jest.mock('../../components/CommitInfo/CommitInfo', () => (props) => {
+//     mockCommitInfo(props);
+//     return <h1>xyzxyz</h1>;
+// });
+
+describe('Commit info page tests', () => {
+    type TestCase = {
+        name: string;
+        commitHash: string;
+        fakeLoadEverything: boolean;
+        commitInfoStoreData:
+            | {
+                  commitInfoReady: CommitInfoState;
+                  response: GetCommitInfoResponse | undefined;
+              }
+            | undefined;
+        expectedSpinnerCount: number;
+        expectedMainContentCount: number;
+        expectedText: string[];
```

Comment:
As long as we don't need an array, make this a `string`.
```suggestion
        expectedText: string;
```